src/app/app.component.ts

```diff
@@ -264,6 +265,8 @@ export class AppComponent {
   }
 
   ngOnInit() {
+    // Ethnio survey 11-10-21
+    this._script.loadScript('ethnio-survey');
```

Comment:
This will be added to every page - recommend removing it. I don't think it needs to be added anywhere else.

Reply:
Ah I see....How would I add it to the item detail page @NazimHAli ?

Reply:
It's already included here : https://github.com/ithaka/aiw-ui/pull/1362/files#diff-416072271f1854db0d48a9b6df3b354d3e35c12f68083ceebe4017a03e5db046L235

Reply:
Does it not show up after you deployed it to test?